spec/lib/translator.js

```diff
@@ -343,4 +344,22 @@ describe('Translator', () => {
             done();
         });
     });
+
+    // Lab will output the amount of time spent in this test. We can use it to
+    // compare relative speed of different implementations.
+    it('load test', { timeout: 5000, skip: true }, done => {
+        fs.readFile('./spec/fixtures/lang.json', 'utf8' , (err, data) => {
+            if (err) {
+                console.error(err);
+                throw err;
+            }
+
+            translations = data;
```

Comment:
Why are you creating another reference to the same object?

Reply:
Yeah that's unnecessary, I can change it